lib/rubocop/cop/mixin/range_help.rb

```diff
@@ -34,6 +34,21 @@ def contents_range(node)
         range_between(node.loc.begin.end_pos, node.loc.end.begin_pos)
       end
 
+      # A range containing the first to the last argument
+      # of a method call or method definition.
+      # def foo(a, b:)
+      #         ^^^^^
+      # bar(1, 2, 3, &blk)
+      #     ^^^^^^^^^^^^^
+      # baz { |x, y:, z: }
+      #        ^^^^^^^^^
+      def arguments_range(node)
+        range_between(
+          node.first_argument.source_range.begin_pos,
+          node.last_argument.source_range.end_pos
+        )
```

Comment:
This could be simplified:

```suggestion
        node.first_argument.source_range.join(node.last_argument.source_range)
```

Reply:
Neat, applied. Thanks